### ISSUE TITLE
[BITV] 9.1.4.11/0.1 - Note: Information-carrying graphics and graphical controls, as well as their states, should have a contrast ratio of at least 3:1 against adjacent colors. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36966]
All pages
https://try.nextcloud.com/ltd/a11y/

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-3dbfb7615a-v0-n1

</details>

As far i've seen there are several places to change:

1. Active state of action menu button:
![Screenshot from 2023-09-22 15-30-12](https://github.com/nextcloud/server/assets/6078378/fbed583f-08b6-4883-8f32-2a5d1892c31b)
- https://github.com/nextcloud-libraries/nextcloud-vue/pull/4576

![Screenshot from 2023-09-22 14-21-31](https://github.com/nextcloud/server/assets/6078378/f634bbc9-ff8f-4817-bc3d-97abcc6b2820)
- https://github.com/nextcloud-libraries/nextcloud-vue/pull/4576

![Screenshot from 2023-09-22 14-02-58](https://github.com/nextcloud/server/assets/6078378/43acbc8a-67cc-49d5-80a8-f91b435780db)

2. Actions on user profile view (opacity have to be removed)
![Screenshot from 2023-09-22 14-05-50](https://github.com/nextcloud/server/assets/6078378/dce1b1aa-1e93-4c62-932e-6b03af992bd2)

![Screenshot from 2023-09-22 14-25-59](https://github.com/nextcloud/server/assets/6078378/6b8c741b-e3c2-4c3e-a6e7-5b3a0be13ca2)

- https://github.com/nextcloud/server/pull/40613

**Comment by JuliaKirschenheuter** [comment]
Dear @michaelnissenbaum 

could you please give us some examples of `Information-carrying graphics and graphical controls`? Which ones are meant there? 
Thank you!

**Comment by JuliaKirschenheuter** [comment]
https://github.com/nextcloud/nextcloud-vue/issues/4193#issuecomment-1600349035

**Comment by JuliaKirschenheuter** [comment]
closing ticket, most found things are done
